documentation/catalog/guides/dag_testing.md

```diff
@@ -0,0 +1,49 @@
+# Running DAGs in development
+
+[Apache Airflow](https://airflow.apache.org/) is used to manage the workflow for
+various API ETL jobs which pull and process data from a number of open APIs on
+the internet. These workflows are otherwise known as DAGs. This document
+describes how to run or test out these DAGs, locally during development.
```

Comment:
It looks like this first line is pulled from our catalog quickstart documentation 😄 Rather than repeat the same text here, maybe we can say something simpler, like `Apache Airflow is used to manage our workflows (DAGs) in the catalog. For more information see [our quickstart guide](/catalog/guides/quickstart.md#api-data)`?